docs/features/sessions/faq.md

```diff
@@ -42,3 +42,9 @@ If prompted for a password while attempting to connect to a Session:
   
   ###grid.ai managed BEGIN do not edit manually### is at the top of the config file.
 :::
+
+## Why does ssh to a Session fail with returned non-zero exit status 255?
+- Make sure that the below line is at the top of the ~/.ssh/config file.
+  
+  ###grid.ai managed BEGIN do not edit manually### is at the top of the config file.
```

Comment:
```suggestion

```

Reply:
@oojo12 do you know what this line is here for?

Reply:
for the ssh config for some reason it apparently needs to be at the top of the file and has been a working solution that @sunitaprakash has prescribed to a few users.

Reply:
Wait what...this doesn't make sense. This is marker for grid CLI which section is managed by us. The users shouldn't have to add these manually. CC @adam-grid

Reply:
It most definitely should not be at the top, we'd delete all users ssh config if that's the case....maybe we should use separate ssh config file and call ssh command with this config file over default one